### PR TITLE
Fix for ARM CI

### PR DESCRIPTION
**Pull request by hcab14**
https://github.com/gnuradio/volk/pull/393

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -73,12 +73,12 @@ matrix:
...
+      env: MATRIX_EVAL="CMAKE_ARG=-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchains/arm-linux-gnueabihf.cmake"
       addons: {apt: {sources: "ubuntu-toolchain-r-test", packages: [*common_packages, g++-arm-linux-gnueabihf, qemu-user]}}
 
     # Job 8 ... ARMv8 (aarch64) cross compile
     - name: Linux ARMv8 (aarch64) Qemu GCC 7
-      env: MATRIX_EVAL="CC=gcc-7 && CXX=g++-7 && CMAKE_ARG=\"-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchains/aarch64-linux-gnu.cmake\""
+      env: MATRIX_EVAL="CMAKE_ARG=-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchains/aarch64-linux-gnu.cmake"
            TOOLCHAIN=LINARO
            TARGET=aarch64-linux-gnu
            QEMU_ARCH=aarch64
```
Comment (jdemel) [inline .travis.yml:84]: I assume these lines can be safely removed? I put them in there not knowing what I did.

**Review by jdemel — APPROVED** [review]
LGTM. Thanks for taking care of this!

There was one job that failed but a rerun fixed it.

File: .travis.yml (hunk 1 above)
Comment (hcab14) [reply .travis.yml:84]: I think so.

**Comment by hcab14** [comment]
> LGTM. Thanks for taking care of this!

It was just an issue of lower case vs. capital case definitions in the cmake toolchain file (cpu-features seems to understand only lower-case architecture names).

> There was one job that failed but a rerun fixed it.

One of the ARM qemu jobs?

**Comment by jdemel** [comment]
> > There was one job that failed but a rerun fixed it.
> 
> One of the ARM qemu jobs?

Yeah. The aarch64 QEMU job failed. one of those 100+ tests only. `volk_32f_x3_sum_of_poly_32f`. This one seems to be flaky sometimes. So I don't worry that much about it.

**Pull request merged**